### ISSUE TITLE
Mapillary pKey in ID Editor einfacher hinzufügen

### ISSUE THREAD

**Issue by vizsim** [osmberlin/osm-traffic-sign-tool#50]
Wenn ich im ID Editor ein Verkehrschild erstelle, möchte ich auch den Mapillary pKey hinzufügen.
Im besten Fall würde der ID Editor gleich `mapillary` als Feld vorschlagern (z.B. wie bei `traffic_sign:direction` "Betroffene Fahrtrichtung")
![grafik](https://github.com/user-attachments/assets/3e439b63-5e2d-4ef5-ac25-506c00718b1c)

Vermutlich ist das eher ein Thema für [id-tagging-schema](https://github.com/openstreetmap/id-tagging-schema), bin aber unsicher wie groß das Rad ist, an dem man da drehen müsste.

Ggf. könntet ihr im osm-traffic-sign-tool auch beim Kopieren eines "Traffic Sign Tag" einfach ein `mapillary=` mitliefern? 
z.B.
```
traffic_sign=DE:240
mapillary=
```

**Comment by tordans** [comment]
Ja, ich habe auch schon überlegt, wie man das besser machen kann.

Ein gutes neues Feature ist: Man kann seit kurzen in iD innerhalb des Mapillary-Fotos ein +-Icon klicken um die ID direkt zuzuordnen.

Zu den Ideen

> Wenn ich im ID Editor ein Verkehrschild erstelle, möchte ich auch den Mapillary pKey hinzufügen.
> Im besten Fall würde der ID Editor gleich mapillary als Feld vorschlagern (z.B. wie bei traffic_sign:direction "Betroffene Fahrtrichtung")
> Vermutlich ist das eher ein Thema für [id-tagging-schema](https://github.com/openstreetmap/id-tagging-schema), bin aber unsicher wie groß das Rad ist, an dem man da drehen müsste.

Technisch kann das id-tagging-schema so etwas schon. Aber eine Hürde wird sein, dass man mit Mapillary einen Dienst bevorzugen würde und Panoramax etc. sich dann melden würden.

Du könntest mal ein Ticket dort machen, ob man das `highway=traffic_sign+traffic_sign=*` Preset erweitern will um die 2-3 verbreiteten Straßenlevel Foto-Dienste…

> Ggf. könntet ihr im osm-traffic-sign-tool auch beim Kopieren eines "Traffic Sign Tag" einfach ein mapillary= mitliefern?
z.B.

Ja, das habe ich vor. Ich benenne das Ticket mal um, weil das das Feature ist, was wir hier im Repo kontrollieren können.

Ich will auch etwas wie `is_sidepath=yes` mit empfehlen. Ich weiß nur noch nicht, wie ich unterscheide zwischen "Etablierte Tags / Quasi Pflicht Tags" und "Sinnvolle Tags die 'in use' sind".
